doc/cookbook/source/examples/classifier/gaussian_naive_bayes.rst

```diff
@@ -0,0 +1,38 @@
+===================================
+Gaussian Naive Bayes Classification
+===================================
+
+Gaussian Naive Bayes classifies data according to how well it aligns with the Gaussian distributions of several different classes.
+
+The probability that some feature :math:`x_{i}` in the feature vector :math:`i` belongs to class :math:`c`, :math:`p(x_{i}|c)`, is given by
+
+.. math::
+
+  p(x_{i}|c)=\frac{1}{\sqrt{2\pi\sigma_{x,c}^{2}}}e^{-\frac{(x_{i}-\mu_{x,c})^{2}}{2\sigma_{x,c}^{2}}}
+  
+For each vector, the Gaussian Naive Bayes classifier chooses the class :math:`c` with maximal
+
+.. math::
+
+  p(c)\prod_{i}p(x_{i}|c)
+
+-------
+Example
+-------
+
+Imagine we have files with training and test data. We create CDenseFeatures (here 64 bit floats aka RealFeatures) and :sgclass:`CMulticlassLabels` as
+
+.. sgexample:: gaussiannaivebayes.sg:create_features
+
+We create an instance of the :sgclass:`CGaussianNaiveBayes` classifier, passing it training data and the label.
+
+.. sgexample:: gaussiannaivebayes.sg:create_instance
+
+Then we run the train Gaussian Naive Bayes algorithm, apply it to the test data, and print the prediction.
```

Comment:
see the updates I made to knn.rst, and also to the Readme. They are minor, but could you please mimic them?
Not printing here. But rather say what comes out of the precition method